### DISCUSSION TITLE
1.5.3.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#634] [Обновления]
## Список изменений

- Исправлено положение субтитров в GoogleDrive (#619)
- Из меню убраны не работающие языки
- Изменена логика работы GM_fetch (#632)

## What's Changed
* Изменение логики GM_fetch by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/632

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.3...1.5.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.3.1'>1.5.3.1</a>.</em>

**Comment by FiLinX** [comment]
Ребят, небольшая просьба, - дополните пожалуйста ещё немножко данные в README.md в Список функционала - описание на все функции что уже наличествуют!
я очень часто даю ссылку на вас в разных форумах и чатах, и русскоговорящим и буржуям, и при этом часто приходится дополнительными предложениями уточнять, что через VOT так же можно выводить и субтитры переведённые яндексом, и то, что и их и саму аудиодорожку перевода - с панели расширения можно скачать себе на комп!
и наверное лучше этот абзац Список функционала поднять повыше..
вобщем, облегчите пожалуйста жизнь и мне и людям сюда направленным) 
чтоб им из описания тут - сразу про весь наличествующий функционал становилось понятно!
Спасибо)

**Reply by ilyhalight** [reply]
Дополнил

**Comment by FiLinX** [comment]
во первых - Спасибо за предыдущее! )
но я с проблемой..
вот ролик - https://youtu.be/CVRWbIMfssw
на котором я стандартно вижу панель с предложением "Перевести видео"  -  https://i.imgur.com/ATouAj0.jpeg
при нажатии, опять же стандартно срабатывает и выводит на панели "Отключить"  -  https://i.imgur.com/LNBaNWL.jpeg
но как только начать воспроизведение, то на панели появляется оранжевая надпись "Формат аудио не поддерживается" - https://i.imgur.com/GjSnPvh.jpeg
ну и естесственно никакого перевода нет..
и так же естесственно нет значка для скачивания дорожки аудио-перевода
при этом звук оригинала становиться приглушенным до уровня как в настройках панели, при клике по оранжевой - звук оригинала нормализуется и на панели снова "Перевести видео"
очень странно.. я постоянно смотрю его ролики, и никогда ещё сбоев с переводом небыло..

**Reply by ilyhalight** [reply]
У Яндекса что-то сломалось и он на том видео решил отдавать невалидную ссылку. Поставь язык на авто и он выдаст нормальную

**Comment by FiLinX** [comment]
скажите, а чего даёт поддержка VOT-ом Invidious-ов?
я почему-то думал, что это нужно для перевода геоблокированных в ютубе роликов, но оказалось это не так..
сегодня вот попался такой геоблокированный - https://www.youtube.com/watch?v=mF9SZ6AULAM 
забиваю его в инвид-лист - https://redirect.invidious.io/watch?v=mF9SZ6AULAM
и начинаю открывать все зеркала 100% здоровые, и с панелькой VOT-а там попались лишь последние два 100%-х - https://i.imgur.com/1BVGUts.jpeg
но млин - "Ошибка: Видео недоступно для перевода"  ((
эхь.. жаль..
видимо без выкача и закача в ядиск не обойтись..)

**Reply by ilyhalight** [reply]
Это больше для тех, кто предпочитает использовать invidious для просмотра вместо ютуба. Перевод там будет точно такой же как на ютубе т.к. ссылка на перевод формируется ютубовская.

> и начинаю открывать все зеркала 100% здоровые, и с панелькой VOT-а там попались лишь последние два 100%-х -

В расширение добавлены только десяток из популярных зеркал, но при желание список можно расширить

**Comment by FiLinX** [comment]
странно - "Нейросети не смогли определить речь на видео"  - https://9gag.com/gag/aRBevEA
даже с вручную выставленным Японский

**Reply by NullVerdict** [reply]
Напишите в поддержку Яндекса

**Comment by pioklo** [comment]
чо-то поломалось... Пропала плашка  с включением перевода вообще.

**Reply by ilyhalight** [reply]
#716
